src/pages/patientView/PatientViewPage.tsx

```diff
@@ -247,6 +275,40 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             });
         }
 
+        if (this.patientIdsInCohort && this.patientIdsInCohort.length > 0) {
```

Comment:
could do that too. I think I have it like this because the other query string parameters (studyId, patientId) are handled like this